dev-libs/c_blosc/c_blosc-1.13.7.recipe

```diff
@@ -0,0 +1,95 @@
+SUMMARY="Blocking, shuffling and lossless compression library"
+DESCRIPTION="Blosc is a high performance compressor optimized for binary \
+data. It has been designed to transmit data to the processor cache faster \
+than the traditional, non-compressed, direct memory fetch approach via a \
+memcpy() OS call.  Blosc is the first compressor (that I'm aware of) that is \
+meant not only to reduce the size of large datasets on-disk or in-memory, but \
+also to accelerate memory-bound computations."
+HOMEPAGE="http://www.blosc.org"
+COPYRIGHT="2012-2017 Francesc Alted"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/Blosc/c-blosc/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="a27005c242a496f7a208fe0a2c6bf2a6956b0d30e87fb1217a5f33e807e90dde"
+SOURCE_DIR="c-blosc-$portVersion"
+PATCHES="c_blosc-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	c_blosc$secondaryArchSuffix = $portVersion compat >= 1
+	lib:libblosc$secondaryArchSuffix = 1.13.7 compat >= 1
```

Comment:
How about using `libVersionCompat`? Also, I think we could just drop `compat` for `c_blosc`, as we only have version >= 1 in HaikuPorts.

```diff
 ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
 PROVIDES="
-	c_blosc$secondaryArchSuffix = $portVersion compat >= 1
-	lib:libblosc$secondaryArchSuffix = 1.13.7 compat >= 1
+	c_blosc$secondaryArchSuffix = $portVersion
+	lib:libblosc$secondaryArchSuffix = $libVersionCompat
 "
```